### PR TITLE
[SW-2504] Python Scoring Package

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

**Review by honzasterba — APPROVED** [review]
out of curiosity, why is spark version no longer part of the package name?

**Comment by mn-mikke** [comment]
@honzasterba Spark version should be a part of package name. Where it's missing? In places like https://github.com/h2oai/sparkling-water/pull/2426/files#diff-cdcb9bc4023f5afb2c1fa4f3961fa2b55b9a517f24ea2fa868e023c5c528135cR23 the Spark verision is ommitted since the jar is a internal file inside the package.

File: ci/Jenkinsfile-release [hunk 1]
```diff
@@ -367,10 +374,10 @@ def publishCondaArtifact(arch, pkgName) {
 }
 
 
-def getCondaPkgName(params, pyVersion) {
+def getCondaPkgName(pyVersion, packageName) {
     return sh(returnStdout: true, script:
             """
-            CONDA_PKG_CURRENT_ARCH_PATH=\$(conda build h2o_pysparkling_${params.spark} --py ${pyVersion} --output-folder "." --output | tail -1)
+            CONDA_PKG_CURRENT_ARCH_PATH=\$(conda build ${packageName} --py ${pyVersion} --output-folder "." --output | tail -1)
```
Comment (honzasterba) [inline ci/Jenkinsfile-release:380]: spark version removed from package name

Comment (mn-mikke) [reply ci/Jenkinsfile-release:380]: see the name entry in packages: https://github.com/h2oai/sparkling-water/pull/2426/files/abbb67db0ab47caa7ed73b3353aa2941ef384ad3#diff-7178e3aa434d15a33bacb11e21cf4af3ce2f2bb6fb692536f75545dc247a8787R280

**Pull request merged**